### PR TITLE
Replace time with std::time

### PR DESCRIPTION
**Pull request by serprex**

### CODE REVIEW

File: src/app.rs [hunk 1]
```diff
@@ -86,7 +85,7 @@ pub fn start(mut conf: Config) {
                 output_folder.push(clone.key);
                 match OpenOptions::new().write(true).append(true).create(true).open(&output_folder)
                     .and_then(|mut file| {
-                        file.write(&format!("{} {}", cur_time, &result).as_bytes()[..])
+                        file.write(&format!("{:?} {}", cur_time, &result).as_bytes()[..])
```
Comment (TheNeikos) [inline src/app.rs]: Will this give the correct output? In this case it should be the Unix Epoch.

**Comment by TheNeikos** [comment]
Heya! Thanks for the PR!

Just have a small question, but otherwise looks good.

File: src/app.rs (hunk 1 above)
Comment (serprex) [inline src/app.rs]: Oh.. Instant is defined purely as a precise monotonic clock. I'll update PR this evening to use [SystemTime](https://doc.rust-lang.org/std/time/struct.SystemTime.html) with [UNIX_EPOCH](https://doc.rust-lang.org/std/time/constant.UNIX_EPOCH.html)

**Comment by serprex** [comment]
Updated

File: src/app.rs [hunk 2]
```diff
@@ -97,7 +100,10 @@ pub fn start(mut conf: Config) {
             });
         }
         if let Some(c) = conf.items.peek() {
-            thread::sleep(Duration::from_secs((c.next_time - get_time().sec) as u64));
+            let cur_time = Instant::now();
+            if cur_time > c.next_time {
+                thread::sleep(c.next_time.duration_since(cur_time));
```
Comment (TheNeikos) [inline src/app.rs]: You have your logic backwards here. You want to know from now to next time.

**Comment by serprex** [comment]
Sorry about the typo; fixed

**Comment by matthiasbeyer** [comment]
IMO this can be merged, right?

**Comment by serprex** [comment]
Presumably

**Comment by TheNeikos** [comment]
Uh, yeah sorry. This flew completely off the radar for me. Gonna check it as soon as I am home

**Comment by mario-kr** [comment]
superseded by rewrite with tokio.

(also, the time-dependency was dropped)

**Pull request closed without merging**